Easydict/Swift/View/MenuItemView.swift

```diff
@@ -67,10 +67,19 @@ struct MenuItemView: View {
                 latestVersion = version
             }
         }
+        .openSettingsAccess()
```

Comment:
According to the doc of `SettingsAcess`, this should not used in menu content.

> If using a menu-based MenuBarExtra, do not apply openSettingsAccess() to the menu content. openSettings() cannot be used there due to limitations of SwiftUI.

https://github.com/orchetect/SettingsAccess?tab=readme-ov-file#2-use-in-a-menubarextra-menu